### PR TITLE
Minor licence update

### PR DESCRIPTION
**Pull request by jculpon**
Sorry if this is reopening old issues, just noticed a typo in my licence. Looked OK on my machine.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2013 Matthew Scott and other contributors.
+Copyright (c) 2013+ Madison Scott and other contributors.
```
Comment (makyo) [inline LICENSE]: Minor: `Madison Scott-Clary`

Comment (jculpon) [inline LICENSE]: Doink. I'll fix it up

**Comment by jculpon** [comment]
Fixed in d35d363

**Comment by jculpon** [comment]
Thanks! :shipit:

\Jamie

**Pull request merged**